caom2/caom2/obs_reader_writer.py

```diff
@@ -73,40 +73,44 @@
 import os
 import uuid
 from builtins import str, int
-from urllib.parse import urlparse
+from enum import Enum
 
 from lxml import etree
 
+import caom2
 from . import artifact
 from . import caom_util
 from . import chunk
+from . import dali
 from . import observation
 from . import part
 from . import plane
 from . import shape
 from . import wcs
-from . import common
 import logging
 
+from .plane import CalibrationStatus, Ucd, Polarization
+
 DATA_PKG = 'data'
 
 CAOM22_SCHEMA_FILE = 'CAOM-2.2.xsd'
```

Comment:
Is this definition still required? Correspondingly, there is a file `caom2/caom2/data/CAOM-2.2.xsd`. Does it need to be kept?